chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+if ! [[ -r `pwd` ]]; then
+    echo "Directory is not readable."
+    exit 255
+elif [[ "$#" -lt 2 ]]; then
+    echo "At least 2 args are expected"
+    exit 1
+elif [[ "$#" -gt 2 ]]; then
+    echo "Too many args, only 2 args are expected"
+    exit 1
+elif [[ -e $1 && -f $2 ]]; then
+    cat $1 $2
```

Comment:
I think it would be better to add some separator between 2 different files contents, now from this output it would be not clear, what is content of the 1st file and what is the 2nd

Reply:
Done